python/src/trezorlib/transport/__init__.py

```diff
@@ -96,21 +105,23 @@ def find_by_path(cls, path: str, prefix_search: bool = False) -> "Transport":
         raise TransportException(f"{cls.PATH_PREFIX} device not found: {path}")
 
 
-def all_transports() -> Iterable[Type[Transport]]:
+def all_transports() -> Iterable[Type["TransportType"]]:
     from .bridge import BridgeTransport
     from .hid import HidTransport
     from .udp import UdpTransport
     from .webusb import WebUsbTransport
 
-    return set(
-        cls
-        for cls in (BridgeTransport, HidTransport, UdpTransport, WebUsbTransport)
-        if cls.ENABLED
+    transports: Tuple[Type["TransportType"], ...] = (
```

Comment:
fwiw, i vaguely remember that `Tuple[Type[Transport], ...]` didn't work in mypy, but maybe it could work here and we could get rid of `TransportType`?

alternately, a little nicer, would this work?
```
T = TypeVar("T", bound=Transport)
transports: Tuple[Type[T], ...]
```

Reply:
I was really able to delete all mentions of `TransportType` and replace it just with `Transport` after defining the `find_debug()` in `Transport`.

```
T = TypeVar("T", bound=Transport)
transports: Tuple[Type[T], ...]
```
did not work, it was complaning about `Expression of type "tuple[Type[BridgeTransport], Type[HidTransport], Type[UdpTransport], Type[WebUsbTransport]]" cannot be assigned to declared type "Tuple[Type[T@all_transports], ...]"` - and also it would mean having to define it in ` if TYPE_CHECKING:`, while simple `Tuple[Type["Transport"], ...]`
 does not even require that